### DISCUSSION TITLE
更新订阅时出现这个，版本是最新的2.10.3
profile does not contain `proxies` or `proxy-providers`

### DISCUSSION THREAD

**Discussion by a131656** [MetaCubeX/ClashMetaForAndroid#301] [Q&A]
![Screenshot_2024-08-28-01-50-14-095_com.github.metacubex.clash.meta-edit.jpg](https://github.com/user-attachments/assets/80b6114e-fdd6-4b80-a8ff-879b56692fd5)

**Comment by woshichenghaibo** [comment]
不知道原因

**Reply by 20041010** [reply]
换一个厂家解决了
